RoddyCore/src/de/dkfz/roddy/config/DerivedFromFilenamePattern.java

```diff
@@ -37,7 +37,7 @@ public FilenamePatternDependency getFilenamePatternDependency() {
 
     @Override
     protected BaseFile getSourceFile(BaseFile[] baseFiles) {
-        return baseFiles[0];
+        return (BaseFile) baseFiles[0].getParentFiles().get(0);
```

Comment:
Yes somehow it is. A workflow is always created with source files. Files without parent files will only be created manually and in very rare cases. So yes it is safe to use.